libs/langchain/langchain/chains/conversational_retrieval/base.py

```diff
@@ -112,6 +115,27 @@ def output_keys(self) -> List[str]:
             _output_keys = _output_keys + ["generated_question"]
         return _output_keys
 
+    @root_validator
+    def validate_combine_docs_chain(cls, values: Dict) -> Dict:
```

Comment:
chat_history is passed into the combine_docs_chain whether or not there's a question_generator. think it's confusing if we change prompts for users, think it's clearer if things error when prompt doesn't support chat_history var